packaging/PLATFORM_SUPPORT.md

```diff
@@ -95,7 +95,10 @@ with minimal user effort.
 | Alpine Linux  | 3.14    | No                       |                                                                                                      |
 | Amazon Linux  | 2       | x86\_64, AArch64         | Scheduled for promotion to Core tier at some point after the release of v1.39.0 of the Netdata Agent |
 | Arch Linux    | Latest  | No                       | We officially recommend the community packages available for Arch Linux                              |
+| Fedora        | 38      | x86\_64, AArch64         | Scheduled for promotion to Core tier at some point after the release of v1.39.0 of the Netdata Agent |
```

Comment:
By general convention, if the distro is already in the core tier for support, then any new releases get added there as well. With limited exceptions like the RHEL 8 fiasco, new releases don’t change things enough that we have to worry about special support schedules for them on our end.

Reply:
I placed those the because we won't produce a stable version until we cut v1.39

Reply:
Move them to _core_